CODEOWNERS

```diff
@@ -124,6 +124,9 @@
 # renovate groupName=buildpack
 /task/build-paketo-builder-oci-ta @cmoulliard
 
+# renovate groupName=oci-run-script
+/task/run-script-oci-ta @Zokormazo
```

Comment:
You can add @konflux-ci/build-maintainers here and group this with the other build tasks :slightly_smiling_face: 

And I think @arewm was also interested in being an owner?